jetbrains-rider/ReSharper.AWS/src/AWS.Daemon/ZoneMarker.cs

```diff
@@ -0,0 +1,11 @@
+using JetBrains.Application.BuildScript.Application.Zones;
+using JetBrains.ReSharper.Feature.Services;
+using JetBrains.ReSharper.Psi.CSharp;
+
+namespace ReSharper.AWS

```

Comment:
What is the purpose of the ZoneMarker classes?

Reply:
This is ReSharper way to resolve component dependencies and specify which components need to be up when launching ReSharper process. 
Here are some details - https://www.jetbrains.com/help/resharper/sdk/Platform/Zones.html

Reply:
Oh, Resharper SDK docs! Will read through this.